tests/expected/check_operations/operations.rs

```diff
@@ -0,0 +1,11 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0 OR MIT
+//
+// Check the message printed when a checked operation fails.
+fn main() {
+    let v1: u8 = kani::any();
+    let v2: u8 = kani::any();
+    let _ = v1 + v2;
+    let _ = v1 - v2;
+    let _ = v1 * v2;
```

Comment:
Add for division as well (with an assumption that v2 is non-zero)?

Reply:
Sure. I can add a few more cases here.